### PR TITLE
Context-aware cv in subgraph queries

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Make sure all subgraph queries either:
- use the cv provided in the current context
- include both V2 and V3 in their cv_in queries

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/pages/projects/index.page.tsx [hunk 1]
```diff
@@ -93,8 +93,8 @@ function Projects() {
     const _cv: CV[] = []
     if (includeV1) _cv.push(CV_V1)
     if (includeV1_1) _cv.push(CV_V1_1)
-    if (includeV2) _cv.push(CV_V2)
-    return _cv.length ? _cv : [CV_V1, CV_V1_1, CV_V2]
+    if (includeV2) _cv.push(CV_V2, CV_V3)
```
Comment (peripheralist) [inline src/pages/projects/index.page.tsx:96]: ohh nice addition. i'm thinking maybe we should change the UI for that version picker to be more explicit about including v3, though that can be another PR

File: src/models/cv.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { V1TerminalVersion } from 'models/v1/terminals'
 export type V2CVType = '2'
 export type V3CVType = '3'
+export type V2V3CVType = V2CVType | V3CVType
```
Comment (peripheralist) [inline src/models/cv.ts:4]: lol i'm realizing these types are a bit redundant—cv" stands for "contracts version", so the extra V isn't needed

these could be `CV2Type`, `CV2V3Type` etc maybe. same thing with the constants names. maybe out of scope for this PR

**Review by peripheralist — APPROVED** [review]
hard to know for sure from the diff if this is _all_ the cv instances that need to be updated, but all changes so far look solid to me! left a few comments you might want to address, but approving and leaving it up to you 🙏

File: src/pages/projects/index.page.tsx (hunk 1 above)
Comment (tomquirk) [reply src/pages/projects/index.page.tsx:96]: yeah totally agree, just went with this for completeness and speed but we should prob add another checkbox 👍

File: src/models/cv.ts (hunk 2 above)
Comment (tomquirk) [reply src/models/cv.ts:4]: Yeah haha, gr8 point indeed.

I think CV2V3Type is tight. will change

Comment (tomquirk) [reply src/models/cv.ts:4]: > V2CVType | V3CVType

Yeah I think I got all these, but will check again 👍
